src/towncrier/create.py

```diff
@@ -59,11 +61,32 @@ def __main(directory, config, filename):
     if os.path.exists(segment_file):
         raise click.ClickException("{} already exists".format(segment_file))
 
+    content = _get_news_content(edit)
+
+    if content is None:
+        click.echo("Abort creating news fragment.")
+        ctx.exit(1)
+
     with open(segment_file, "w") as f:
-        f.writelines(["Add your info here"])
+        f.write(content)
 
     click.echo("Created news fragment at {}".format(segment_file))
 
 
+def _get_news_content(edit, default_content="Add your info here"):
+    if not edit:
+        return default_content
```

Comment:
I think this would be simpler to read if the condition was moved outside the function call.  There's just not enough logic or code to need to be 'abstracted'.